src/didactWebView.ts

```diff
@@ -47,24 +47,24 @@ export class DidactWebviewPanel {
 		DidactWebviewPanel.context = ctxt;
 	}
 
-	public getMDPath() {
-		return this.mdPath;
+	public getDidactUriPath() {
+		return this.didactUriPath;
 	}
 
-	public setMarkdown(value: string | undefined) {
-		this.mdStr = value;
+	public setDidactStr(value: string | undefined) {
+		this.didactStr = value;
 	}
 
 	public getCurrentHTML() : string | undefined {
 		return this.currentHtml;
 	}
 
 	getMarkdown() {
-		return this.mdStr;
+		return this.didactStr;
```

Comment:
is it a didactStr that can be used for both Markdown and asciidoc or a markdown one?

Reply:
Will look

Reply:
Good catch. It should be used for both.